canonical-web-and-design/maas-ui
# fix(ui): fix mismatched LXD list columns on smaller screens

## Done

- Fixed incorrect classname for the zone column in the LXD list

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the LXD list and check that the column headers and content match up for all screen sizes

## Fixes

Fixes #3215
